model/project_parser.go

```diff
@@ -505,15 +505,14 @@ func LoadProjectForVersion(v *Version, id string, shouldSave bool) (ProjectInfo,
 	var pp *ParserProject
 	var err error
 
-	pRef, err := FindMergedProjectRef(id, "", false)
-	if err != nil {
-		return ProjectInfo{}, errors.Wrap(err, "error finding project ref")
-	}
 	pp, err = ParserProjectFindOneById(v.Id)
 	if err != nil {
 		return ProjectInfo{}, errors.Wrap(err, "error finding parser project")
 	}
-
+	pc, err := FindProjectConfig(v.Identifier, v.Id)
```

Comment:
Why do we no longer use version control here?

Reply:
I did this because this is a DB call and should be quick, and isn't doing any unmarshalling like in other places we check for VC. To check for VC we need to add a DB query to retrieve the project ref so I thought we may as well just retrieve the project config if we are going to add 1 DB query.  That being said I'm happy to use version control here for consistency

Reply:
That's a good point. I guess it's up to us to decide what the version control flag means. Does it mean that if it's on, we save and use project configs, and will still use it for any version if it exists even if we toggle the flag off? This would mean we only need to check it in the places where we save the project config. Or does it mean that if it's off, it's off and we don't use the config even if we've stored it? 

Since we _did_ initially create this flag to make DB improvements I would actually be fine to have it mean the former thing. Whatever we decide, we just need to make sure it's documented. And we may want one more ticket to fix things for whichever behavior we decide and/or verify the behavior.

Reply:
Yeah I think it makes most sense for when it's off, don't use the config whatsoever even if we've stored it. I think it's more intuitive.